Wikipedia.xcodeproj/xcshareddata/xcschemes/Wikipedia.xcscheme

```diff
@@ -92,7 +92,8 @@
       ignoresPersistentStateOnLaunch = "NO"
       debugDocumentVersioning = "YES"
       debugServiceExtension = "internal"
-      allowLocationSimulation = "YES">
+      allowLocationSimulation = "YES"
```

Comment:
please revert this. it might be easier to use the dedicated RTL scheme to avoid issues w/ making the regular scheme RTL